src/adapter-types/src/timestamp_selection.rs

```diff
@@ -20,13 +20,13 @@ pub enum ConstraintBasedTimestampSelection {
 
 impl Default for ConstraintBasedTimestampSelection {
     fn default() -> Self {
-        Self::Verify
+        Self::Enabled
     }
 }
 
 impl ConstraintBasedTimestampSelection {
     pub const fn const_default() -> Self {
-        Self::Verify
+        Self::Enabled
```

Comment:
@frankmcsherry Is this fine?

Reply:
I'm positive, but @DAlperin might be the best person to confirm that this is low risk.

Reply:
Dov confirms via Slack